quick-start/src/main/ui/app/jobs/jobs.component.html

```diff
@@ -60,6 +65,10 @@
                   <i (click)="showTraces(job.jobId)" class="fa fa-flash"></i>
                 </mdl-button>
               </td>
+              <td class="job-delete-checkbox">
+                <input type="checkbox" id="list-checkbox-{{job.jobId}}" class="mdl-checkbox__input"
```

Comment:
Check out mdl-checkbox: http://mseemann.io/angular2-mdl/toggle

It's prettier.